src/vzlogger.cpp

```diff
@@ -549,6 +581,13 @@ int main(int argc, char *argv[]) {
 					::sleep(1); // 1s should be ok. will introduce a shutdown latency >1s
 				}
 			}
+			if (mainLoopReopenLogfile) {
+				mainLoopReopenLogfile = false;
+				print(log_info, "closing logfile for re-opening (requested with SIGUSR1)", "");
+				closeLogfile(true);
```

Comment:
that's fine. You could use =false as default value.
but here the 
m_log.lock();
close...(true)
open...(true)
m_log.unlock();
is needed then, or?

Reply:
ofcourse.
i was so annoyed by implementing this that i didn't care to do it right.
fixed.

Reply:
:-) thx.

Reply:
added default false, not so familiar with C++ and didn't occur to me that it can do this.